ng/src/web/pages/radius/radiuspage.js

```diff
@@ -0,0 +1,171 @@
+/* Greenbone Security Assistant
+*
+* Authors:
+* Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+*
+* Copyright:
+* Copyright (C) 2017 Greenbone Networks GmbH
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+
+import _ from 'gmp/locale.js';
+import {YES_VALUE, NO_VALUE} from 'gmp/parser.js';
+
+import Button from '../../components/form/button.js';
+import CheckBox from '../../components/form/checkbox.js';
+import FormGroup from '../../components/form/formgroup.js';
+import PasswordField from '../../components/form/passwordfield.js';
+import TextField from '../../components/form/textfield.js';
+
+import HelpIcon from '../../components/icon/helpicon.js';
+
+import Layout from '../../components/layout/layout.js';
+import Section from '../../components/section/section.js';
+
+import Loading from '../../components/loading/loading.js';
+
+import PropTypes from '../../utils/proptypes.js';
+
+import withGmp from '../../utils/withGmp.js';
+
+
+class RadiusAuthentication extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      enable: '',
+      radiushost: '',
+      radiuskey: '',
+      loading: 'true',
+    }
+
+    this.getRadiusAuth = this.getRadiusAuth.bind(this);
```

Comment:
Just a hint: It's not required to bind this.getRadiusAuth because it is not passed as a handler.